tensorflow_datasets/structured/adult_test.py

```diff
@@ -0,0 +1,23 @@
+"""Test for UCI Adult Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.structured import adult
+
+
+class AdultTest(testing.DatasetBuilderTestCase):
+  
+  DATASET_CLASS = adult.Adult
+  SPLITS = {
+      "test": 5
```

Comment:
Please include tests for other splits. Usually one file per split is good enough.